<!DOCTYPE html>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
<meta http-equiv="x-ua-compatible" content="IE=edge"/>
<title>Test results - Class com.commercetools.pspadapter.payone.PaymentHandlerTest</title>
<link href="../css/base-style.css" rel="stylesheet" type="text/css"/>
<link href="../css/style.css" rel="stylesheet" type="text/css"/>
<script src="../js/report.js" type="text/javascript"></script>
</head>
<body>
<div id="content">
<h1>Class com.commercetools.pspadapter.payone.PaymentHandlerTest</h1>
<div class="breadcrumbs">
<a href="../index.html">all</a> &gt; 
<a href="../packages/com.commercetools.pspadapter.payone.html">com.commercetools.pspadapter.payone</a> &gt; PaymentHandlerTest</div>
<div id="summary">
<table>
<tr>
<td>
<div class="summaryGroup">
<table>
<tr>
<td>
<div class="infoBox" id="tests">
<div class="counter">9</div>
<p>tests</p>
</div>
</td>
<td>
<div class="infoBox" id="failures">
<div class="counter">4</div>
<p>failures</p>
</div>
</td>
<td>
<div class="infoBox" id="ignored">
<div class="counter">0</div>
<p>ignored</p>
</div>
</td>
<td>
<div class="infoBox" id="duration">
<div class="counter">2.366s</div>
<p>duration</p>
</div>
</td>
</tr>
</table>
</div>
</td>
<td>
<div class="infoBox failures" id="successRate">
<div class="percent">55%</div>
<p>successful</p>
</div>
</td>
</tr>
</table>
</div>
<div id="tabs">
<ul class="tabLinks">
<li>
<a href="#tab0">Failed tests</a>
</li>
<li>
<a href="#tab1">Tests</a>
</li>
<li>
<a href="#tab2">Standard output</a>
</li>
</ul>
<div id="tab0" class="tab">
<h2>Failed tests</h2>
<div class="test">
<a name="returnsStatusCodeInternalServerError500InCaseOfUnexpectedExceptionFromPaymentDispatcher"></a>
<h3 class="failures">returnsStatusCodeInternalServerError500InCaseOfUnexpectedExceptionFromPaymentDispatcher</h3>
<span class="code">
<pre>java.lang.AssertionError: 
Expecting:
 &lt;&quot;Unexpected error occurred when processing commercetools Payment with id [1504832329]. See the service logs&quot;&gt;
to contain:
 &lt;&quot;Unexpected error occurred when processing payment [1504832329]. See the service logs&quot;&gt; 
	at com.commercetools.pspadapter.payone.PaymentHandlerTest.returnsStatusCodeInternalServerError500InCaseOfUnexpectedExceptionFromPaymentDispatcher(PaymentHandlerTest.java:245)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.mockito.internal.junit.JUnitRule$1.evaluate(JUnitRule.java:16)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.runTestClass(JUnitTestClassExecuter.java:114)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.execute(JUnitTestClassExecuter.java:57)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassProcessor.processTestClass(JUnitTestClassProcessor.java:66)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:51)
	at sun.reflect.GeneratedMethodAccessor60.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:32)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
	at com.sun.proxy.$Proxy1.processTestClass(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.processTestClass(TestWorker.java:108)
	at sun.reflect.GeneratedMethodAccessor59.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:146)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:128)
	at org.gradle.internal.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:404)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
	at java.lang.Thread.run(Thread.java:748)
</pre>
</span>
</div>
<div class="test">
<a name="returnsStatusCodeInternalServerError500InCaseOfUnexpectedExceptionFromPaymentWithCartLike"></a>
<h3 class="failures">returnsStatusCodeInternalServerError500InCaseOfUnexpectedExceptionFromPaymentWithCartLike</h3>
<span class="code">
<pre>java.lang.AssertionError: 
Expecting:
 &lt;&quot;Unexpected error occurred when processing commercetools Payment with id [1989869346]. See the service logs&quot;&gt;
to contain:
 &lt;&quot;Unexpected error occurred when processing payment [1989869346]. See the service logs&quot;&gt; 
	at com.commercetools.pspadapter.payone.PaymentHandlerTest.returnsStatusCodeInternalServerError500InCaseOfUnexpectedExceptionFromPaymentWithCartLike(PaymentHandlerTest.java:224)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.mockito.internal.junit.JUnitRule$1.evaluate(JUnitRule.java:16)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.runTestClass(JUnitTestClassExecuter.java:114)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.execute(JUnitTestClassExecuter.java:57)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassProcessor.processTestClass(JUnitTestClassProcessor.java:66)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:51)
	at sun.reflect.GeneratedMethodAccessor60.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:32)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
	at com.sun.proxy.$Proxy1.processTestClass(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.processTestClass(TestWorker.java:108)
	at sun.reflect.GeneratedMethodAccessor59.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:146)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:128)
	at org.gradle.internal.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:404)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
	at java.lang.Thread.run(Thread.java:748)
</pre>
</span>
</div>
<div class="test">
<a name="returnsStatusCodeNotFound404InCaseOfCartLikeMissing"></a>
<h3 class="failures">returnsStatusCodeNotFound404InCaseOfCartLikeMissing</h3>
<span class="code">
<pre>java.lang.AssertionError: 
Expecting:
 &lt;&quot;Failed to process the commercetools Payment with id [646755414], as the payment or the cart could not be found.&quot;&gt;
to contain:
 &lt;&quot;Error on processing payment with ID [646755414] as payment or cart could not be found&quot;&gt; 
	at com.commercetools.pspadapter.payone.PaymentHandlerTest.returnsStatusCodeNotFound404InCaseOfCartLikeMissing(PaymentHandlerTest.java:169)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.mockito.internal.junit.JUnitRule$1.evaluate(JUnitRule.java:16)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.runTestClass(JUnitTestClassExecuter.java:114)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.execute(JUnitTestClassExecuter.java:57)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassProcessor.processTestClass(JUnitTestClassProcessor.java:66)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:51)
	at sun.reflect.GeneratedMethodAccessor60.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:32)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
	at com.sun.proxy.$Proxy1.processTestClass(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.processTestClass(TestWorker.java:108)
	at sun.reflect.GeneratedMethodAccessor59.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:146)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:128)
	at org.gradle.internal.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:404)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
	at java.lang.Thread.run(Thread.java:748)
</pre>
</span>
</div>
<div class="test">
<a name="returnsStatusCodeNotFound404InCaseOfUnknownPayment"></a>
<h3 class="failures">returnsStatusCodeNotFound404InCaseOfUnknownPayment</h3>
<span class="code">
<pre>java.lang.AssertionError: 
Expecting:
 &lt;&quot;Failed to process the commercetools Payment with id [1617519906], as the payment or the cart could not be found.&quot;&gt;
to contain:
 &lt;&quot;Error on processing payment with ID [1617519906] as payment or cart could not be found&quot;&gt; 
	at com.commercetools.pspadapter.payone.PaymentHandlerTest.returnsStatusCodeNotFound404InCaseOfUnknownPayment(PaymentHandlerTest.java:152)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.mockito.internal.junit.JUnitRule$1.evaluate(JUnitRule.java:16)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.runTestClass(JUnitTestClassExecuter.java:114)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.execute(JUnitTestClassExecuter.java:57)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassProcessor.processTestClass(JUnitTestClassProcessor.java:66)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:51)
	at sun.reflect.GeneratedMethodAccessor60.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:32)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
	at com.sun.proxy.$Proxy1.processTestClass(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.processTestClass(TestWorker.java:108)
	at sun.reflect.GeneratedMethodAccessor59.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:146)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:128)
	at org.gradle.internal.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:404)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
	at java.lang.Thread.run(Thread.java:748)
</pre>
</span>
</div>
</div>
<div id="tab1" class="tab">
<h2>Tests</h2>
<table>
<thead>
<tr>
<th>Test</th>
<th>Duration</th>
<th>Result</th>
</tr>
</thead>
<tr>
<td class="success">returnsStatusCodeAccepted202InCaseOfOngoingConcurrentPaymentHandling</td>
<td>2.012s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">returnsStatusCodeBadRequest400InCaseOfPaymentWithOtherPaymentInterface</td>
<td>0.007s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">returnsStatusCodeInternalServerError500InCaseOfUnexpectedExceptionFromCommercetoolsQuery</td>
<td>0.014s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="failures">returnsStatusCodeInternalServerError500InCaseOfUnexpectedExceptionFromPaymentDispatcher</td>
<td>0.002s</td>
<td class="failures">failed</td>
</tr>
<tr>
<td class="failures">returnsStatusCodeInternalServerError500InCaseOfUnexpectedExceptionFromPaymentWithCartLike</td>
<td>0.001s</td>
<td class="failures">failed</td>
</tr>
<tr>
<td class="failures">returnsStatusCodeNotFound404InCaseOfCartLikeMissing</td>
<td>0.001s</td>
<td class="failures">failed</td>
</tr>
<tr>
<td class="failures">returnsStatusCodeNotFound404InCaseOfUnknownPayment</td>
<td>0.020s</td>
<td class="failures">failed</td>
</tr>
<tr>
<td class="success">returnsStatusCodeOk200InCaseOfSuccessfulConcurrentPaymentHandling</td>
<td>0.308s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">returnsStatusCodeOk200InCaseOfSuccessfulPaymentHandling</td>
<td>0.001s</td>
<td class="success">passed</td>
</tr>
</table>
</div>
<div id="tab2" class="tab">
<h2>Standard output</h2>
<span class="code">
<pre>{&quot;timestamp&quot;:&quot;2019-12-18T09:46:23.422+00:00&quot;,&quot;loggerName&quot;:&quot;com.commercetools.pspadapter.payone.PaymentHandler[tenant testTenantName]&quot;,&quot;severity&quot;:&quot;ERROR&quot;,&quot;message&quot;:&quot;Failed to process the commercetools Payment with id [1617519906], as the payment or the cart could not be found.&quot;,&quot;tenantName&quot;:&quot;testTenantName&quot;,&quot;stack_hash&quot;:&quot;15eb79ef&quot;,&quot;stack_trace&quot;:&quot;i.s.s.c.NotFoundException: http response: &lt;unknown&gt;\nSDK: 1.48.0\nproject: &lt;unknown&gt;\nJava: 1.8.0_151\ncwd: /home/travis/build/commercetools/commercetools-payone-integration/service\nsphere request: &lt;unknown&gt;\nadditional notes: []\nJavadoc: http://commercetools.github.io/commercetools-jvm-sdk/apidocs/io/sphere/sdk/client/NotFoundException.html\n\n\tat o.j.r.ParentRunner.runLeaf(ParentRunner.java:325)\n\tat o.j.r.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)\n\tat o.j.r.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)\n\tat o.j.r.ParentRunner$3.run(ParentRunner.java:290)\n\tat o.j.r.ParentRunner$1.schedule(ParentRunner.java:71)\n\tat o.j.r.ParentRunner.runChildren(ParentRunner.java:288)\n\tat o.j.r.ParentRunner.access$000(ParentRunner.java:58)\n\tat o.j.r.ParentRunner$2.evaluate(ParentRunner.java:268)\n\tat o.j.r.ParentRunner.run(ParentRunner.java:363)\n\tat o.g.a.i.t.t.j.JUnitTestClassExecuter.runTestClass(JUnitTestClassExecuter.java:114)\n\tat o.g.a.i.t.t.j.JUnitTestClassExecuter.execute(JUnitTestClassExecuter.java:57)\n\tat o.g.a.i.t.t.j.JUnitTestClassProcessor.processTestClass(JUnitTestClassProcessor.java:66)\n\tat o.g.a.i.t.t.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:51)\n\tat s.r.GeneratedMethodAccessor60.invoke(Unknown Source)\n\tat s.r.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat j.l.reflect.Method.invoke(Method.java:498)\n\tat o.g.i.d.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)\n\tat o.g.i.d.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)\n\tat o.g.i.d.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:32)\n\tat o.g.i.d.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)\n\tat c.sun.proxy.$Proxy1.processTestClass(Unknown Source)\n\tat o.g.a.i.t.t.w.TestWorker.processTestClass(TestWorker.java:108)\n\tat s.r.GeneratedMethodAccessor59.invoke(Unknown Source)\n\tat s.r.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat j.l.reflect.Method.invoke(Method.java:498)\n\tat o.g.i...&quot;}
{&quot;timestamp&quot;:&quot;2019-12-18T09:46:23.433+00:00&quot;,&quot;loggerName&quot;:&quot;com.commercetools.pspadapter.payone.PaymentHandler[tenant testTenantName]&quot;,&quot;severity&quot;:&quot;WARN&quot;,&quot;message&quot;:&quot;Exception on dispatchPayment with id [395048094]. Retry 1 of 20&quot;,&quot;tenantName&quot;:&quot;testTenantName&quot;}
{&quot;timestamp&quot;:&quot;2019-12-18T09:46:23.534+00:00&quot;,&quot;loggerName&quot;:&quot;com.commercetools.pspadapter.payone.PaymentHandler[tenant testTenantName]&quot;,&quot;severity&quot;:&quot;WARN&quot;,&quot;message&quot;:&quot;Exception on dispatchPayment with id [395048094]. Retry 2 of 20&quot;,&quot;tenantName&quot;:&quot;testTenantName&quot;}
{&quot;timestamp&quot;:&quot;2019-12-18T09:46:23.634+00:00&quot;,&quot;loggerName&quot;:&quot;com.commercetools.pspadapter.payone.PaymentHandler[tenant testTenantName]&quot;,&quot;severity&quot;:&quot;WARN&quot;,&quot;message&quot;:&quot;Exception on dispatchPayment with id [395048094]. Retry 3 of 20&quot;,&quot;tenantName&quot;:&quot;testTenantName&quot;}
{&quot;timestamp&quot;:&quot;2019-12-18T09:46:23.746+00:00&quot;,&quot;loggerName&quot;:&quot;com.commercetools.pspadapter.payone.PaymentHandler[tenant testTenantName]&quot;,&quot;severity&quot;:&quot;ERROR&quot;,&quot;message&quot;:&quot;Completion exception error: com.commercetools.pspadapter.payone.PaymentHandlerTest$1: 1540300425\nCause: null&quot;,&quot;tenantName&quot;:&quot;testTenantName&quot;}
{&quot;timestamp&quot;:&quot;2019-12-18T09:46:23.753+00:00&quot;,&quot;loggerName&quot;:&quot;com.commercetools.pspadapter.payone.PaymentHandler[tenant testTenantName]&quot;,&quot;severity&quot;:&quot;WARN&quot;,&quot;message&quot;:&quot;Wrong payment interface name: expected [TestIntegrationServicePaymentMethodName], found [other than TestIntegrationServicePaymentMethodName]&quot;,&quot;tenantName&quot;:&quot;testTenantName&quot;}
{&quot;timestamp&quot;:&quot;2019-12-18T09:46:23.766+00:00&quot;,&quot;loggerName&quot;:&quot;com.commercetools.pspadapter.payone.PaymentHandler[tenant testTenantName]&quot;,&quot;severity&quot;:&quot;WARN&quot;,&quot;message&quot;:&quot;Exception on dispatchPayment with id [563908566]. Retry 1 of 20&quot;,&quot;tenantName&quot;:&quot;testTenantName&quot;}
{&quot;timestamp&quot;:&quot;2019-12-18T09:46:23.869+00:00&quot;,&quot;loggerName&quot;:&quot;com.commercetools.pspadapter.payone.PaymentHandler[tenant testTenantName]&quot;,&quot;severity&quot;:&quot;WARN&quot;,&quot;message&quot;:&quot;Exception on dispatchPayment with id [563908566]. Retry 2 of 20&quot;,&quot;tenantName&quot;:&quot;testTenantName&quot;}
{&quot;timestamp&quot;:&quot;2019-12-18T09:46:23.969+00:00&quot;,&quot;loggerName&quot;:&quot;com.commercetools.pspadapter.payone.PaymentHandler[tenant testTenantName]&quot;,&quot;severity&quot;:&quot;WARN&quot;,&quot;message&quot;:&quot;Exception on dispatchPayment with id [563908566]. Retry 3 of 20&quot;,&quot;tenantName&quot;:&quot;testTenantName&quot;}
{&quot;timestamp&quot;:&quot;2019-12-18T09:46:24.070+00:00&quot;,&quot;loggerName&quot;:&quot;com.commercetools.pspadapter.payone.PaymentHandler[tenant testTenantName]&quot;,&quot;severity&quot;:&quot;WARN&quot;,&quot;message&quot;:&quot;Exception on dispatchPayment with id [563908566]. Retry 4 of 20&quot;,&quot;tenantName&quot;:&quot;testTenantName&quot;}
{&quot;timestamp&quot;:&quot;2019-12-18T09:46:24.170+00:00&quot;,&quot;loggerName&quot;:&quot;com.commercetools.pspadapter.payone.PaymentHandler[tenant testTenantName]&quot;,&quot;severity&quot;:&quot;WARN&quot;,&quot;message&quot;:&quot;Exception on dispatchPayment with id [563908566]. Retry 5 of 20&quot;,&quot;tenantName&quot;:&quot;testTenantName&quot;}
{&quot;timestamp&quot;:&quot;2019-12-18T09:46:24.270+00:00&quot;,&quot;loggerName&quot;:&quot;com.commercetools.pspadapter.payone.PaymentHandler[tenant testTenantName]&quot;,&quot;severity&quot;:&quot;WARN&quot;,&quot;message&quot;:&quot;Exception on dispatchPayment with id [563908566]. Retry 6 of 20&quot;,&quot;tenantName&quot;:&quot;testTenantName&quot;}
{&quot;timestamp&quot;:&quot;2019-12-18T09:46:24.371+00:00&quot;,&quot;loggerName&quot;:&quot;com.commercetools.pspadapter.payone.PaymentHandler[tenant testTenantName]&quot;,&quot;severity&quot;:&quot;WARN&quot;,&quot;message&quot;:&quot;Exception on dispatchPayment with id [563908566]. Retry 7 of 20&quot;,&quot;tenantName&quot;:&quot;testTenantName&quot;}
{&quot;timestamp&quot;:&quot;2019-12-18T09:46:24.471+00:00&quot;,&quot;loggerName&quot;:&quot;com.commercetools.pspadapter.payone.PaymentHandler[tenant testTenantName]&quot;,&quot;severity&quot;:&quot;WARN&quot;,&quot;message&quot;:&quot;Exception on dispatchPayment with id [563908566]. Retry 8 of 20&quot;,&quot;tenantName&quot;:&quot;testTenantName&quot;}
{&quot;timestamp&quot;:&quot;2019-12-18T09:46:24.571+00:00&quot;,&quot;loggerName&quot;:&quot;com.commercetools.pspadapter.payone.PaymentHandler[tenant testTenantName]&quot;,&quot;severity&quot;:&quot;WARN&quot;,&quot;message&quot;:&quot;Exception on dispatchPayment with id [563908566]. Retry 9 of 20&quot;,&quot;tenantName&quot;:&quot;testTenantName&quot;}
{&quot;timestamp&quot;:&quot;2019-12-18T09:46:24.672+00:00&quot;,&quot;loggerName&quot;:&quot;com.commercetools.pspadapter.payone.PaymentHandler[tenant testTenantName]&quot;,&quot;severity&quot;:&quot;WARN&quot;,&quot;message&quot;:&quot;Exception on dispatchPayment with id [563908566]. Retry 10 of 20&quot;,&quot;tenantName&quot;:&quot;testTenantName&quot;}
{&quot;timestamp&quot;:&quot;2019-12-18T09:46:24.772+00:00&quot;,&quot;loggerName&quot;:&quot;com.commercetools.pspadapter.payone.PaymentHandler[tenant testTenantName]&quot;,&quot;severity&quot;:&quot;WARN&quot;,&quot;message&quot;:&quot;Exception on dispatchPayment with id [563908566]. Retry 11 of 20&quot;,&quot;tenantName&quot;:&quot;testTenantName&quot;}
{&quot;timestamp&quot;:&quot;2019-12-18T09:46:24.872+00:00&quot;,&quot;loggerName&quot;:&quot;com.commercetools.pspadapter.payone.PaymentHandler[tenant testTenantName]&quot;,&quot;severity&quot;:&quot;WARN&quot;,&quot;message&quot;:&quot;Exception on dispatchPayment with id [563908566]. Retry 12 of 20&quot;,&quot;tenantName&quot;:&quot;testTenantName&quot;}
{&quot;timestamp&quot;:&quot;2019-12-18T09:46:24.973+00:00&quot;,&quot;loggerName&quot;:&quot;com.commercetools.pspadapter.payone.PaymentHandler[tenant testTenantName]&quot;,&quot;severity&quot;:&quot;WARN&quot;,&quot;message&quot;:&quot;Exception on dispatchPayment with id [563908566]. Retry 13 of 20&quot;,&quot;tenantName&quot;:&quot;testTenantName&quot;}
{&quot;timestamp&quot;:&quot;2019-12-18T09:46:25.073+00:00&quot;,&quot;loggerName&quot;:&quot;com.commercetools.pspadapter.payone.PaymentHandler[tenant testTenantName]&quot;,&quot;severity&quot;:&quot;WARN&quot;,&quot;message&quot;:&quot;Exception on dispatchPayment with id [563908566]. Retry 14 of 20&quot;,&quot;tenantName&quot;:&quot;testTenantName&quot;}
{&quot;timestamp&quot;:&quot;2019-12-18T09:46:25.173+00:00&quot;,&quot;loggerName&quot;:&quot;com.commercetools.pspadapter.payone.PaymentHandler[tenant testTenantName]&quot;,&quot;severity&quot;:&quot;WARN&quot;,&quot;message&quot;:&quot;Exception on dispatchPayment with id [563908566]. Retry 15 of 20&quot;,&quot;tenantName&quot;:&quot;testTenantName&quot;}
{&quot;timestamp&quot;:&quot;2019-12-18T09:46:25.274+00:00&quot;,&quot;loggerName&quot;:&quot;com.commercetools.pspadapter.payone.PaymentHandler[tenant testTenantName]&quot;,&quot;severity&quot;:&quot;WARN&quot;,&quot;message&quot;:&quot;Exception on dispatchPayment with id [563908566]. Retry 16 of 20&quot;,&quot;tenantName&quot;:&quot;testTenantName&quot;}
{&quot;timestamp&quot;:&quot;2019-12-18T09:46:25.374+00:00&quot;,&quot;loggerName&quot;:&quot;com.commercetools.pspadapter.payone.PaymentHandler[tenant testTenantName]&quot;,&quot;severity&quot;:&quot;WARN&quot;,&quot;message&quot;:&quot;Exception on dispatchPayment with id [563908566]. Retry 17 of 20&quot;,&quot;tenantName&quot;:&quot;testTenantName&quot;}
{&quot;timestamp&quot;:&quot;2019-12-18T09:46:25.475+00:00&quot;,&quot;loggerName&quot;:&quot;com.commercetools.pspadapter.payone.PaymentHandler[tenant testTenantName]&quot;,&quot;severity&quot;:&quot;WARN&quot;,&quot;message&quot;:&quot;Exception on dispatchPayment with id [563908566]. Retry 18 of 20&quot;,&quot;tenantName&quot;:&quot;testTenantName&quot;}
{&quot;timestamp&quot;:&quot;2019-12-18T09:46:25.575+00:00&quot;,&quot;loggerName&quot;:&quot;com.commercetools.pspadapter.payone.PaymentHandler[tenant testTenantName]&quot;,&quot;severity&quot;:&quot;WARN&quot;,&quot;message&quot;:&quot;Exception on dispatchPayment with id [563908566]. Retry 19 of 20&quot;,&quot;tenantName&quot;:&quot;testTenantName&quot;}
{&quot;timestamp&quot;:&quot;2019-12-18T09:46:25.675+00:00&quot;,&quot;loggerName&quot;:&quot;com.commercetools.pspadapter.payone.PaymentHandler[tenant testTenantName]&quot;,&quot;severity&quot;:&quot;WARN&quot;,&quot;message&quot;:&quot;Exception on dispatchPayment with id [563908566]. Retry 20 of 20&quot;,&quot;tenantName&quot;:&quot;testTenantName&quot;}
{&quot;timestamp&quot;:&quot;2019-12-18T09:46:25.775+00:00&quot;,&quot;loggerName&quot;:&quot;com.commercetools.pspadapter.payone.PaymentHandler[tenant testTenantName]&quot;,&quot;severity&quot;:&quot;ERROR&quot;,&quot;message&quot;:&quot;The payment [563908566] couldn't be processed after 20 retries&quot;,&quot;tenantName&quot;:&quot;testTenantName&quot;}
{&quot;timestamp&quot;:&quot;2019-12-18T09:46:25.777+00:00&quot;,&quot;loggerName&quot;:&quot;com.commercetools.pspadapter.payone.PaymentHandler[tenant testTenantName]&quot;,&quot;severity&quot;:&quot;ERROR&quot;,&quot;message&quot;:&quot;Error in response: &quot;,&quot;tenantName&quot;:&quot;testTenantName&quot;,&quot;stack_hash&quot;:&quot;3467676e&quot;,&quot;stack_trace&quot;:&quot;j.l.RuntimeException: 1584595604\n\tat o.j.r.ParentRunner.runLeaf(ParentRunner.java:325)\n\tat o.j.r.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)\n\tat o.j.r.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)\n\tat o.j.r.ParentRunner$3.run(ParentRunner.java:290)\n\tat o.j.r.ParentRunner$1.schedule(ParentRunner.java:71)\n\tat o.j.r.ParentRunner.runChildren(ParentRunner.java:288)\n\tat o.j.r.ParentRunner.access$000(ParentRunner.java:58)\n\tat o.j.r.ParentRunner$2.evaluate(ParentRunner.java:268)\n\tat o.j.r.ParentRunner.run(ParentRunner.java:363)\n\tat o.g.a.i.t.t.j.JUnitTestClassExecuter.runTestClass(JUnitTestClassExecuter.java:114)\n\tat o.g.a.i.t.t.j.JUnitTestClassExecuter.execute(JUnitTestClassExecuter.java:57)\n\tat o.g.a.i.t.t.j.JUnitTestClassProcessor.processTestClass(JUnitTestClassProcessor.java:66)\n\tat o.g.a.i.t.t.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:51)\n\tat s.r.GeneratedMethodAccessor60.invoke(Unknown Source)\n\tat s.r.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat j.l.reflect.Method.invoke(Method.java:498)\n\tat o.g.i.d.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)\n\tat o.g.i.d.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)\n\tat o.g.i.d.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:32)\n\tat o.g.i.d.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)\n\tat c.sun.proxy.$Proxy1.processTestClass(Unknown Source)\n\tat o.g.a.i.t.t.w.TestWorker.processTestClass(TestWorker.java:108)\n\tat s.r.GeneratedMethodAccessor59.invoke(Unknown Source)\n\tat s.r.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat j.l.reflect.Method.invoke(Method.java:498)\n\tat o.g.i.d.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)\n\tat o.g.i.d.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)\n\tat o.g.i.r.i.h.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:146)\n\tat o.g.i.r.i.h.MessageHubBackedObjectConnection$DispatchWrappe...&quot;}
</pre>
</span>
</div>
</div>
<div id="footer">
<p>
<div>
<label class="hidden" id="label-for-line-wrapping-toggle" for="line-wrapping-toggle">Wrap lines
<input id="line-wrapping-toggle" type="checkbox" autocomplete="off"/>
</label>
</div>Generated by 
<a href="http://www.gradle.org">Gradle 4.5.1</a> at Dec 18, 2019 9:50:52 AM</p>
</div>
</div>
</body>
</html>
